ale/drivers/mex_drivers.py

```diff
@@ -422,3 +495,85 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 1
+
+class MexHrscIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, RadialDistortion, Driver):
+
+        @property
+        def instrument_id(self):
+            if(super().instrument_id != "HRSC"):
+                raise Exception ("Instrument ID is wrong.")
+
+            return super().instrument_id
+
+        @property
+        def sensor_model_version(self):
+            return 1
+
+        @property
+        def line_scan_rate(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
+            start_time = times_table['EphemerisTime'][0]
+            return times_table['LineStart'], times_table['EphemerisTime'], times_table['ExposureTime']
+
+        @property
+        def ephemeris_start_time(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
+            start_time = times_table['EphemerisTime'][0]
+            return start_time
+
+        @property
+        def ephemeris_stop_time(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
```

Comment:
Calling these two methods at the start of each methods is clunky. Make these into properties that store the value, similar to the PDS driver.